Description/el-GR.md

```diff
@@ -0,0 +1,84 @@
+# Ελληνικά (Ελλάδας)
+
+Ένα πακέτο τροποποιήσεων βασισμένο στο Fabric, το οποίο στοχεύει σε βελτιώσεις απόδοσης και γραφικών.
+
+#### Αναθεώριση του πακέτου στα Αγγλικά:
```

Comment:
```suggestion
Αναθεώριση του πακέτου στα Αγγλικά:
```

I don't see a need for that header format here, unless you want to propose a new row for English as well.

Reply:
It just makes it more visible idk if u want I can remove it

Reply:
Or should I bold the text?

Reply:
My point is that if there is a highlighted text in one language, it should be in all.